base/applications/utilman/process.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * PROJECT:         ReactOS Utility Manager (Accessibility)
+ * LICENSE:         GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:         Process handling functions
+ * COPYRIGHT:       Copyright 2019 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+/* INCLUDES *******************************************************************/
+
+#include "precomp.h"
+
+/* FUNCTIONS ******************************************************************/
+
+/**
+ * @IsProcessRunning
+ * @implemented
+ *
+ * Checks if a process is running.
+ *
+ * @param ProcName
+ * The name of the executable process (in constant WCHARs).
+ *
+ * @return
+ * Returns TRUE if the given process' name is running,
+ * FALSE otherwise.
+ *
+ */
+BOOL IsProcessRunning(IN LPCWSTR lpProcessName)
+{
+    /* Create a snapshot and check if the given process' executable name is running */
+    HANDLE hSnapshot = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0);
+    PROCESSENTRY32W Process = {0};
+    Process.dwSize = sizeof(Process);
+
+    if (Process32FirstW(hSnapshot, &Process))
+    {
+        do
+        {
+            if (lstrcmpW(Process.szExeFile, lpProcessName) == 0)
+            {
+                /*
+                    If the given process' executable name is running then free the handle from
+                    memory and return TRUE.
+                */
+                CloseHandle(hSnapshot);
+                return TRUE;
+            }
+        }
+        /* Go to the next process until we find it */
+        while (Process32NextW(hSnapshot, &Process));
+    }
+
+    /* Process is not running, free the handle and bail out */
+    CloseHandle(hSnapshot);
+    return FALSE;
+}
+
+/**
+ * @LaunchProcess
+ * @implemented
+ *
+ * Executes a process.
+ *
+ * @param lpProcessName
+ * The name of the executable process (in constant wide string).
+ *
+ * @return
+ * Returns TRUE if the process has been launched successfully,
+ * FALSE otherwise.
+ *
+ */
+BOOL LaunchProcess(LPCWSTR lpProcessName)
+{
+    INT Success;
```

Comment:
You can either replace INT by INT_PTR (should be better) or use an HINSTANCE directly.